builtin/credential/github/backend.go

```diff
@@ -11,6 +11,8 @@ import (
 	"golang.org/x/oauth2"
 )
 
+const operationPrefixGitHub = "git-hub"
```

Comment:
I remember deciding on snake case for these, but why is `github` itself hyphenated?

Reply:
b/c it would evaluate to `GitHub` (?)

the snake case languages would evaluate to `git_hub` (correct?).
my vote is for one word, `Github` should be fine

Reply:
Yeap I was going with their official capitalization, but github is ok too imo

Reply:
Updated to `github`